<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnitTestDialog</class>
 <widget class="QDialog" name="UnitTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BuzzCoin</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="cursor">
          <cursorShape>IBeamCursor</cursorShape>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;BuzzCoin&lt;/span&gt; UnitTest&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="presetComboBox"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>5</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pBClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>.SF Compact Text</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhMultiLine</set>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="l1B">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>V1b:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le1B">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
           <property name="text">
            <string>1000000000</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="l1E">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>V1e:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le1E">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
           <property name="text">
            <string>22000000000</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="l1S">
           <property name="text">
            <string>V1s:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le1S">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
           <property name="inputMask">
            <string/>
           </property>
           <property name="text">
            <string>1000000000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb1">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="l2B">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>V2b:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le2B">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
           <property name="text">
            <string>800000</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="l2E">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>V2e:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le2E">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
           <property name="text">
            <string>999999</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="l2S">
           <property name="text">
            <string>V2s:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le2S">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>10000</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb2">
           <property name="text">
            <string/>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="l3B">
           <property name="minimumSize">
            <size>
             <width>25</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>V3b:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le3B">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="l3E">
           <property name="minimumSize">
            <size>
             <width>28</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>V3e:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le3E">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhDigitsOnly</set>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="l3S">
           <property name="text">
            <string>V3s:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le3S">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="cb3">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="pB1">
         <property name="text">
          <string>CoinYearReward</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pB2">
         <property name="text">
          <string>POS Reward</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pB3">
         <property name="text">
          <string>POW Reward</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pB4">
         <property name="text">
          <string>MinStakeAge</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="pBDone">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pBDone</sender>
   <signal>rejected()</signal>
   <receiver>UnitTestDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBDone</sender>
   <signal>accepted()</signal>
   <receiver>UnitTestDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pB1</sender>
   <signal>clicked()</signal>
   <receiver>UnitTestDialog</receiver>
   <slot>test1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>177</x>
     <y>508</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pB2</sender>
   <signal>clicked()</signal>
   <receiver>UnitTestDialog</receiver>
   <slot>test2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pB3</sender>
   <signal>clicked()</signal>
   <receiver>UnitTestDialog</receiver>
   <slot>test3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>478</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pB4</sender>
   <signal>clicked()</signal>
   <receiver>UnitTestDialog</receiver>
   <slot>test4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>482</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>504</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBClear</sender>
   <signal>clicked()</signal>
   <receiver>UnitTestDialog</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>564</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>470</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb1</sender>
   <signal>stateChanged(int)</signal>
   <receiver>UnitTestDialog</receiver>
   <slot>stateChangedCb2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>589</x>
     <y>369</y>
    </hint>
    <hint type="destinationlabel">
     <x>610</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb2</sender>
   <signal>stateChanged(int)</signal>
   <receiver>UnitTestDialog</receiver>
   <slot>stateChangedCb1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>595</x>
     <y>404</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb3</sender>
   <signal>stateChanged(int)</signal>
   <receiver>UnitTestDialog</receiver>
   <slot>stateChangedCb3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>588</x>
     <y>444</y>
    </hint>
    <hint type="destinationlabel">
     <x>611</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>test1()</slot>
  <slot>test2()</slot>
  <slot>test3()</slot>
  <slot>test4()</slot>
  <slot>clear()</slot>
  <slot>stateChangedCb1()</slot>
  <slot>stateChangedCb2()</slot>
  <slot>stateChangedCb3()</slot>
 </slots>
</ui>
